BAC0/core/devices/Points.py

```diff
@@ -734,7 +724,12 @@ def __repr__(self):
                 )
             )
             # Probably disconnected
-            val = None
+            return "{}/{} : (n/a) {}".format(
```

Comment:
needed to duplicate the string formatting here because it's an error to use `{:.2f}` on `None`